src/shared/utils/markdown.ts

```diff
@@ -0,0 +1,19 @@
+import { Marked } from 'marked';
+import { gfmHeadingId } from 'marked-gfm-heading-id';
+import { IOptions } from 'sanitize-html';
+import cleanText from './cleanText';
+
+const options = { async: false } as const;
+
+const marked = new Marked(options, gfmHeadingId());
+
+export const parseMarkdown = (
```

Comment:
Easier single function to use, instead of having to call marked and sanitize-html separately.
Also, recent updates to marked meant more complex options (async: false) and we now need an extra dependency to have ids added to heading (important for links to specific sections of help pages) so this new dependency is used here (marked-gfm-heading-id)